src/Zio/FileSystems/ReadOnlyFileSystem.cs

```diff
@@ -102,7 +102,11 @@ protected override Stream OpenFileImpl(UPath path, FileMode mode, FileAccess acc
     protected override FileAttributes GetAttributesImpl(UPath path)
     {
         // All paths are readonly
-        return base.GetAttributesImpl(path) | FileAttributes.ReadOnly;
+        var attributes = base.GetAttributesImpl(path);
+
+        return attributes == FileAttributes.Normal
+            ? FileAttributes.ReadOnly
```

Comment:
Breaking change: Normal is used when there are no attributes. Combining "Normal" and "ReadOnly" isn't valid, so I changed it so it only returns ReadOnly.